### PR TITLE
Adelle/dynamic flow

### PR DESCRIPTION
**Pull request by Adelle-Pitsas**
- Groups nodes and reorients creation of nodes so many can be added and top node is the first in the stack. Now able to dynamically render many nodes and move them around together
- Moves flow route to home page
- Sets up local storage but not fully implemented @jacobheric I'll ask you about this tomorrow morning! But punting so I can move onto more important functionality
- Sets up modal route with updated serviceMap info

Example of stacked and grouped nodes able to move together: 
Screenshot 2023-08-02 at 10 34 45 AM

### CODE REVIEW

File: islands/flow.tsx [hunk 1]
```diff
@@ -35,15 +39,68 @@ export type Node = {
...
 };
 
+const flowKey: string = "flow-storage";
+
 export default function Flow({ data }: { data: GetServiceMapResponse }) {
   const [nodes, setNodes, onNodesChange] = useNodesState([]);
   const [edges, setEdges, onEdgeChange] = useEdgesState([]);
+  const [rfInstance, setRfInstance] = useState<any>();
+
+  const example = [
```
Comment (Adelle-Pitsas) [inline islands/flow.tsx:49]: Test seed data I'm keeping in here so I can test multiple nodes

File: islands/flow.tsx [hunk 2]
```diff
@@ -113,32 +175,77 @@ export default function Flow({ data }: { data: GetServiceMapResponse }) {
     setEdges(newEdges);
   }, [nodes]);
 
+  useEffect(() => {
+    if (nodes) {
+      console.log(edges);
+      console.log("the fuck is this", rfInstance);
+      setStorage();
```
Comment (Adelle-Pitsas) [inline islands/flow.tsx:182]: Currently saving to local storage but not doing anything with it yet. Not 100% sure how to implement but not sure its that important for this week

File: islands/flow.tsx [hunk 3]
```diff
@@ -113,32 +175,77 @@ export default function Flow({ data }: { data: GetServiceMapResponse }) {
...
+          },
+        };
+      },
+    );
+    return [consumerNodes, producerNodes].flat();
   };
 
+  const setStorage = useCallback(() => {
+    if (rfInstance) {
+      console.log("the fuck is this", rfInstance);
```
Comment (jacobheric) [inline islands/flow.tsx:243]: 😬

File: routes/rules.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { Layout } from "../components/layout.tsx";
```
Comment (jacobheric) [inline routes/rules.tsx:1]: You are welcome to simply delete this as it's now obsolete.

**Review by jacobheric — APPROVED** [review]

**Pull request merged**